docs/code_formatting_rules.md

```diff
@@ -145,7 +145,7 @@ value, err := bar(
 
 **Note that the above guidelines don't apply to log or error messages.** For
 log and error messages, committers should attempt to minimize the number of
-lines utilized, while still adhering to the 80-character column limit. For
+lines utilized, while still adhering to the 80-character column limit For
```

Comment:
dont think this is correct. Either leave as is, or replace the `. For` with `, for`

Reply:
Should be fixed up now!